ipsec_offload_plugin/ipsec_offload/ipsec_offload.c

```diff
@@ -909,31 +910,23 @@ METHOD(kernel_ipsec_t, del_policy, status_t,
 			     "add entry failed err_code[ %d]", err);
 
 		if (data->sa->mode == MODE_TUNNEL) {
-			err = ipsec_outer_ipv4_decap_mod_table(IPSEC_TABLE_DEL,
```

Comment:
Can you share the latest link for P4. This link  doesn't work. https://github.com/intel-innersource/networking.ethernet.acceleration.vswitch.p4-sde.dataplane-p4/blob/kamalaka_rebase_ipsec/mev_reference_p4_programs/fxp-net-scenarios/fxp-net_linux-networking-v3/fxp-net_linux-networking-v3.p4

Reply:
Code has been merged to the main. P4 can be found here - https://github.com/intel-innersource/networking.ethernet.acceleration.vswitch.p4-sde.dataplane-p4/blob/master/mev_reference_p4_programs/fxp-net-scenarios/fxp-net_linux-networking-v3/fxp-net_linux-networking-v3.p4